personal-library/routes/api.js

```diff
@@ -1,126 +1,100 @@
 /*
-*
-*
-*       FILL IN EACH ROUTE BELOW COMPLETELY
-*       
-*       
-*/
+ *
+ *
+ *       Complete the API routing below
+ *
+ *
+ */
 
-'use strict';
+"use strict";
 
-const expect = require('chai').expect;
-const MongoClient = require('mongodb').MongoClient;
-const ObjectId = require('mongodb').ObjectId;
-const MONGODB_CONNECTION_STRING = process.env.DB_URI;
+const MongoClient = require("mongodb").MongoClient;
+const ObjectId = require("mongodb").ObjectId;
+const MONGODB_CONNECTION_STRING = process.env.DB;
+const Mongoose = require("mongoose");
+const BookModel = require("../db/BookModel").BookModel;
+//Example connection: MongoClient.connect(MONGODB_CONNECTION_STRING, function(err, db) {});
 
-module.exports = function (app) {
+Mongoose.connect(MONGODB_CONNECTION_STRING, (err) => {
```

Comment:
To handle the two deprecation warnings:
```suggestion
Mongoose.connect(MONGODB_CONNECTION_STRING,  { useNewUrlParser: true, useUnifiedTopology: true }, (err) => {
```

Reply:
Whoops! Meant to go back and add this.